src/mobileJoystick.js

```diff
@@ -0,0 +1,105 @@
+// Create an SVG element
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
+const circleInside = document.createElementNS(
+  'http://www.w3.org/2000/svg',
+  'circle'
+)
+
+// Set the width as a percentage of the screen size
+const screenWidth = window.innerWidth
+const svgWidth = (20 / 100) * screenWidth
+const joystickRadius = svgWidth / 2
+const joystickNippleRadius = svgWidth / 8
+
+export const isMobileDevice = {
```

Comment:
Please rename `isMobileDevice` -> `mobileJoystick`.

`isMobileDevice` is a name for a boolean value, which would tell if user is playing from mobile device or not.
It was for another idea, sorry for confusion :)